tests/hooks/test_invoke_lsp_glob_guard.py

```
@@ -0,0 +1,427 @@
+    def test_extension_only_has_no_symbols(self):
+        assert guard.find_glob_symbols("src/**/*.ts") == []
+
+    def test_lowercase_concept_has_no_symbols(self):
+        assert guard.find_glob_symbols("*auth*") == []
+
+    def test_short_token_has_no_symbols(self):
+        assert guard.find_glob_symbols("*Foo*") == []
+
+    def test_zero_width_stripped_before_detection(self):
+        # A zero-width space inside the symbol must not hide it from detection.
+        assert guard.find_glob_symbols("*User​Service*") == ["UserService"]
```

Ruff (0.15.14)

[error] Invalid unescaped character zero-width-space, use "\u200B" instead

Replace with escape sequence

(PLE2515)